Source/cmake/OptionsGTK.cmake

```diff
@@ -238,8 +239,8 @@ elseif (WEBKITGTK_API_VERSION VERSION_EQUAL "4.1")
     CALCULATE_LIBRARY_VERSIONS_FROM_LIBTOOL_TRIPLE(WEBKIT 2 0 2)
     CALCULATE_LIBRARY_VERSIONS_FROM_LIBTOOL_TRIPLE(JAVASCRIPTCORE 2 0 2)
 elseif (WEBKITGTK_API_VERSION VERSION_EQUAL "5.0")
-    CALCULATE_LIBRARY_VERSIONS_FROM_LIBTOOL_TRIPLE(WEBKIT 0 0 0)
-    CALCULATE_LIBRARY_VERSIONS_FROM_LIBTOOL_TRIPLE(JAVASCRIPTCORE 0 0 0)
+    CALCULATE_LIBRARY_VERSIONS_FROM_LIBTOOL_TRIPLE(WEBKIT 1 0 1)
+    CALCULATE_LIBRARY_VERSIONS_FROM_LIBTOOL_TRIPLE(JAVASCRIPTCORE 1 0 1)
```

Comment:
Why did you bump library version here?

Reply:
It's in case this PR is ready to land before https://github.com/WebKit/WebKit/pull/3834 is ready. That PR is still blocked by https://github.com/WebKit/WebKit/pull/3858 (which is awaiting your review, when you have time for it), whereas this PR is ready with no other dependencies.

Basically the goal is that once this PR lands, I can start making API changes.

Reply:
BTW when #3834 lands, I will reset this back to `0 0 0` since it would be a new API name.

Also, I considered submitting a separate pull request to get rid of CALCULATE_LIBRARY_VERSIONS_FROM_LIBTOOL_TRIPLE, since using libtool triples just seems rather confusing to me. But since you and Adrian make the releases, it's really only your opinions that matter for that. Curious what you think.